src/transformers/configuration_utils.py

```diff
@@ -784,6 +784,13 @@ def to_diff_dict(self) -> Dict[str, Any]:
             ):
                 serializable_config_dict[key] = value
 
+        if hasattr(self, "quantization_config"):
```

Comment:
This should also only be added if `"quantization_config"` doesn't exist in `default_config_dict`, or isn't the default value. At the moment it's fine, but it's not robust to changes upstream in `PretrainedConfig()`